pkg/capabilities/pb/registry.proto

```diff
@@ -16,20 +16,54 @@ message RemoteTriggerConfig {
   google.protobuf.Duration batchCollectionPeriod = 6;
 }
 
+// deprecated - v1 only
 message RemoteTargetConfig {
   // A collection of dot seperated paths to attributes that should be excluded from the request sent to the remote target
   // when calculating the hash of the request.  This is useful for excluding attributes that are not deterministic to ensure
   // that the hash of logically identical requests is consistent.
   repeated string requestHashExcludedAttributes = 1;
 }
 
+enum TransmissionSchedule {
+  AllAtOnce = 0;
+  OneAtATime = 1;
+}
+
+enum RequestHasherType {
+  Simple = 0;
+  WriteReportExcludeSignatures = 1;
+}
+
+enum AggregatorType {
+  Mode = 0;
+  SignedReport = 1;
+}
+
 message RemoteExecutableConfig {
   // A collection of dot seperated paths to attributes that should be excluded from the request sent to the remote executable capability
   // when calculating the hash of the request.  This is useful for excluding attributes that are not deterministic to ensure
   // that the hash of logically identical requests is consistent.
-  repeated string requestHashExcludedAttributes = 1;
-  google.protobuf.Duration registrationRefresh = 2;
-  google.protobuf.Duration registrationExpiry = 3;
+  repeated string requestHashExcludedAttributes = 1; // deprecated - v1 only
+  reserved 2;  // unused registrationRefresh
+  reserved 3;  // unused registrationExpiry
+
+  TransmissionSchedule transmission_schedule = 4;
+  google.protobuf.Duration delta_stage = 5;
+  google.protobuf.Duration request_timeout = 6;
+  uint32 server_max_parallel_requests = 7;
+  RequestHasherType request_hasher_type = 8;
+}
+
+message AggregatorConfig {
+  AggregatorType aggregator_type = 1;
+}
+
+message CapabilityMethodConfig {
+  oneof remote_config {
```

Comment:
this will require a custom unmarshall for the local CRE like this one https://github.com/smartcontractkit/chainlink/pull/19207, which makes me wonder if what I did for the "other" oneOf field makes sense once we merge this

Reply:
That is only for JSON config in Local CRE, right? Can we reuse the same logic here?